docs/ga-requirements.md

```diff
@@ -0,0 +1,51 @@
+P0 (e.g. cannot GA without these):
+* End-to-end tests
+  * OTLP, Jaeger and Zipkin
+  * Tomcat, Wildfly, Spring Boot
```

Comment:
This list seems sort of arbitrary. Is it P0 to test 3 servlet implementations? My suggestion based on back-of-hand estimation of popularity nowadays is one servlet, spring-boot, and grpc, while others could be P1 or P2

Reply:
Spring Boot certainly covers Tomcat or Jetty, which is all the same. Wildfly is useful, because JBoss is tricky classloader-wise. It uses its own module system and is very sensitive to logging infra problems.

I think current smoke tests selection, Boot, Wildfly and Play, is quite good and propose to continue with that

Reply:
Wildfly is problematic for us, because of both jboss modules (class loading) and jboss logging (we need avoid touching java.util.logging before it loads).

I think our normal instrumentation tests cover Tomcat, Spring Boot and gRPC decently, so I would be good with pushing them to lower priority.

```suggestion
  * Wildfly, due to common regressions around jboss modules (class loading) and jboss logging (we need avoid touching java.util.logging before it loads)
```